debian/rules.in

```diff
@@ -114,6 +118,10 @@ override_dh_installdocs-indep:
 	mv debian/linuxcnc-doc-fr/usr/share/doc/@MAIN_PACKAGE_NAME@/gcode.html debian/linuxcnc-doc-fr/usr/share/doc/@MAIN_PACKAGE_NAME@/gcode_fr.html
 	mv debian/linuxcnc-doc-fr/usr/share/doc/@MAIN_PACKAGE_NAME@ debian/linuxcnc-doc-fr/usr/share/doc/linuxcnc
 	
+	dh_installdocs --doc-main-package=linuxcnc-uspace --package=linuxcnc-doc-nb
+	mv debian/linuxcnc-doc-nb/usr/share/doc/linuxcnc-uspace/gcode.html debian/linuxcnc-doc-nb/usr/share/doc/linuxcnc-uspace/gcode_de.html
```

Comment:
This is a typo.  Should it be nb/gcode.html, or gcode_nb.html?

Reply:
Fixed.